composer.json

```diff
@@ -17,10 +17,10 @@
             "type": "package",
             "package": {
                 "name": "danskernesdigitalebibliotek/dpl-react",
-                "version": "0.2.1",
+                "version": "0.2.0-rc.5",
                 "type": "drupal-library",
                 "dist": {
-                    "url": "https://github.com/danskernesdigitalebibliotek/dpl-react/releases/download/release-feature%2Fsearch-result-app/dist.zip",
+                    "url": "https://github.com/reload/dpl-react/releases/download/release-feature%2Fautocomplete-accessibility/dist.zip",
```

Comment:
Please update this to point to the official builds.

We should also drop ad16c28034b77a78d1038ede96d1d60aee24dc19 as the related fixes are available in builds of `release/3` in the official repo.